dev/CommonStyles/CommandBar_themeresources.xaml

```diff
@@ -813,7 +814,7 @@
                               XYFocusKeyboardNavigation="Enabled">
                             <Grid.ColumnDefinitions>
                                 <ColumnDefinition Width="*" />
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="Auto" MinWidth="{StaticResource AppBarRightMargin}"/>
```

Comment:
>AppBarRightMargin [](start = 89, length = 17)

How is this a margin?

Reply:
It's a right margin -- basically, if the MoreButton isn't visible, it ensures a right margin between the command buttons and the edge of the control. But I can rename it if that's the issue.

Reply:
OK, renamed to AppBarMoreButtonColumnMinWidth.